building/tracks/config-json.md

```diff
@@ -238,7 +238,7 @@ Each concept is an entry in the top-level `concepts` array. The following fields
 ## Key features
 
 The language's key features succinctly describe what the most important features of the language are.
-They are intended to upsell the more interesting features of a language to potential students.
+They are intended to outsell the more interesting features of a language to potential students.
```

Comment:
```suggestion
They are intended to promote the more interesting features of a language to potential students.
```

Maybe "promote" or "advertize" here.

"outsell" is incorrect:

> Definition of outsell
> transitive verb

> 1: archaic : to exceed in value
> 2: to exceed in number of items sold
> 3: to surpass in selling or salesmanship

> Examples of outsell in a sentence:
> - Diet sodas are now beginning to outsell regular sodas in the supermarkets.
> - We've been outselling our competitors for the past five years.


"upsell" was probably incorrect too:
> Definition of upsell
> transitive verb

> to try to convince (a customer) to purchase something additional or at a higher cost

> Examples of upsell in a sentence:
> - With a strong correlation between the amount of video consumed and the Internet speed that consumers think they need, providers could possibly upsell customers on costlier broadband packages.
> - They start upselling you before you book your vacation by offering upgrades to a higher-class cabin